pkg/driver/node_test.go

```diff
@@ -547,34 +547,6 @@ func TestNodeUnstageVolume(t *testing.T) {
 				expectErr(t, err, codes.InvalidArgument)
 			},
 		},
-		{
-			name: "fail GetDeviceName returns error",
-			testFunc: func(t *testing.T) {
-				mockCtl := gomock.NewController(t)
-				defer mockCtl.Finish()
-
-				mockMetadata := cloud.NewMockMetadataService(mockCtl)
-				mockMounter := NewMockMounter(mockCtl)
-				mockDeviceIdentifier := NewMockDeviceIdentifier(mockCtl)
-
-				awsDriver := &nodeService{
-					metadata:         mockMetadata,
-					mounter:          mockMounter,
-					deviceIdentifier: mockDeviceIdentifier,
-					inFlight:         internal.NewInFlight(),
-				}
-
-				mockMounter.EXPECT().GetDeviceNameFromMount(gomock.Eq(targetPath)).Return("", 0, errors.New("GetDeviceName faield"))
```

Comment:
Why do we no longer expect this to fail?

Reply:
We do not return a failure from `NodeUnstageVolume` if `GetDeviceNameFromMount` does not find a device, see https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1526#discussion_r1135577592.